### PR TITLE
Chore: Update lint and corresponding files.

### PR DESCRIPTION
**Pull request by ryan-roemer**
Doing **only** lint fixes and impacted code changes to scope this as narrowly as possible. Part of #2236

### CODE REVIEW

File: packages/victory-bar/src/v37/victory-bar.stories.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable react/no-multi-comp */
 import * as React from "react";
 import VictoryBar from "./victory-bar";
-import VictoryChart from "victory-chart/lib/v37/victory-chart";
+import VictoryChart from "victory-chart/es/v37/victory-chart";
```
Comment (ryan-roemer) [inline packages/victory-bar/src/v37/victory-bar.stories.tsx:4]: Storybook uses webpack which uses our `es/**` files not `lib/**`. Our wireit config thus depends on `build:lib:esm` so this is the proper refererence.

**Review by scottrippey — APPROVED** [review]
Everything looks good to me.  Ran locally without issues.

**Review by hnji — COMMENTED** [review]
N

File: packages/victory-bar/src/v37/victory-bar.stories.tsx (hunk 1 above)
Comment (hnji) [reply packages/victory-bar/src/v37/victory-bar.stories.tsx:4]: - [ ] Yo

**Pull request merged**